salt/state.py

```diff
@@ -1936,7 +1936,13 @@ def event(self, chunk_ret, length, fire_event=False):
         chunk is evaluated an event will be set up to the master with the
         results.
         '''
-        if not self.opts.get('local') and (self.opts.get('state_events', True) or fire_event) and self.opts.get('master_uri'):
+        if not self.opts.get('local') and (self.opts.get('state_events', True) or fire_event):
```

Comment:
I'm somewhat doubtful that this and the following conditions are sufficient for determining whether the state is being applied on the master or the minion. I'm looking for insight on whether this is a good change or not.